### ISSUE TITLE
compliance reporting profiles counts should NOT change when filtering by status

### ISSUE THREAD

**Issue by vjeffrey** [chef/automate#3178]
## Describe the bug
When calling the compliance/reporting/profiles API, counts are returned. These counts will be used for the filter buttons in the UI.
It appears that these counts are being modified when a status filter is included in the request.
```
"counts":{"total":6,"failed":4,"skipped":2,"passed":0,"waived":0}}
                                                                                                                           
curl -s --insecure -H "api-token: $ACCEPTANCE_TOKEN" https://a2-local-inplace-upgrade-acceptance.cd.chef.co/api/v0/compliance/reporting/profiles -d '{"filters":[{"type":"status","values":["passed"]}]}'
{"profiles":[],"counts":{"total":0,"failed":0,"skipped":0,"passed":0,"waived":0}}
```

## To Reproduce
- call the compliance reporting profiles api. note counts.
- call the same api with a status filter. note difference in counts.

## Expected behavior
The counts returned with the API should be unaffected by a status filter. 
In other words, if i call the API with no filters, I should get a full list of profiles and the counts.
If I call the API with a status filter, I should get a list of profiles that have the same status as the requested one, but the counts should remain unchanged.
This will allow us to use backend filtering for the status filters on that page.
